oops/hosts/voyager/iss.py

```
@@ -18,8 +18,7 @@
 ################################################################################
 # Standard class methods
 ################################################################################
-
-def from_file(filespec, astrometry=False, action='error', parameters={}):
+def from_file(filespec, astrometry=False, action='error', method='strict', parameters={}):
```

Ruff (0.15.12)

[warning] Too many branches (20 > 12)

(PLR0912)